externs/google_visualization_api.js

```diff
@@ -0,0 +1,480 @@
+// Copyright 2009 Google Inc.
+// All Rights Reserved.
```

Comment:
Just checking that you got permission to use this file?
Blockly Games is Apache 2, and this file is not.